store/models.py

```diff
@@ -30,3 +30,10 @@ def __str__(self):
         else:
             return f'{self.book.title} - Available'
 
+class BookRating(models.Model):
+    book = models.ForeignKey(Book, on_delete=models.CASCADE)
+    rater = models.CharField(max_length=122)
+    rated = models.BooleanField(default = False)
+    rating = models.SmallIntegerField(default=0)
```

Comment:
You can also use `MinValueValidator`, `MaxValueValidator on this field to have database level rules for it.

Reply:
Ok I'll take care of this from now onwards.